saleor/urls.py

```diff
@@ -21,15 +22,20 @@
 
 handler404 = 'saleor.core.views.handle_404'
 
-urlpatterns = [
+non_translateable_urlpatterns = [
```

Comment:
Typo, shuold be `non_translatable_urlpatterns`. The same below with `translatable_urlpatterns`.